Config/config.xml

```diff
@@ -25,6 +25,12 @@
             <tag name="kernel.event_subscriber" />
             <argument type="service" id="request"/>
         </service>
+
+        <service id="thelia.form_validator" class="ReCaptcha\FormValidator\ReCaptchaFormValidator">
```

Comment:
I think this is dangerous to do this because this will not work if another module override the thelia form validator

Reply:
True, but it make the module usable almost out of the box, instead of having to code something.
Additionally, the module could not be used easily for base Thelia forms. This change make it usable in these forms too.

Reply:
Yes I understand your PR and this is very great but like I said it's dangerous because you can think you are protected but you are not because another module override the form validator.
Maybe we can add a warning message in this case ? (I don't know if we can check if another module override this service)

Reply:
Nowadays, ReCaptcha is a must-have Thelia module (it **should** be included directly in Thelia) so any solution for making module's integration easier would be appreciated 👍

Reply:
Sorry we are **not** going to include a module who is connected to a proprietary service in Thelia. 
I agreed we need a more simple way to add this module to a form but like I explain this PR is not a good solution,    
the service can be override by another module, so it will be inefficient and if a bot remove the field `g-recaptcha-response` the module will not check the captcha and will be inefficient too.